packages/store-ui/src/Carousel/ArrowLeft.tsx

```diff
@@ -6,12 +6,7 @@ import SliderArrowLeft from '../Slider/ArrowLeft'
 type Props = ComponentProps<typeof SliderArrowLeft>
 
 const CarouselArrowLeft: FC<Props> = (props) => (
-  <SliderArrowLeft
-    backgroundColor="transparent"
-    color="black"
-    aria-label="Previous Carousel Image"
-    {...props}
-  />
+  <SliderArrowLeft aria-label="Previous Carousel Image" {...props} />
```

Comment:
put aria-label to come from the prop too

Reply:
Default props with `-` character can't be passed to `aria-label` prop.

Reply:
Also, if aria-label is passed through props, it will override.